examples/specs/bar_aggregate_transform.vl.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please run `yarn run build:example bar_aggregate_transform` so the compiled files are included.

Reply:
Done.